### DISCUSSION TITLE
Is it possible to have multiple editor windows?

### DISCUSSION THREAD

**Discussion by vojtaholik** [codesandbox/sandpack#115] [Q&A] [answered]
Is it possible to have multiple shared instances of editor for each file in the project instead of using tabs? :) Similar to split view in `react-ace`: https://github.com/securingsincity/react-ace/blob/master/docs/Split.md
<img src="https://user-images.githubusercontent.com/25487857/139573202-aabb161d-0a7c-4e9b-9b5c-5faebfd3fc30.png" alt="multiple editor windows example" width="600">

**Comment by danilowoz** [comment, marked as answer]
Hey @vojtaholik, unfortunately, we still don't support this, due to the editor only update the code change from the active path (that you see in the tab). But, this is totally doable and I have added your feature request to our internal backlog which we use to prioritize what to build next. We can't make any promises about the timing, but it's on the list!

If you'd like to work on this feature with us, please create a pull request and we'll help you along the way. Basically, we need to the following files:
- https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/hooks/useActiveCode.ts#L10
- https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/components/CodeEditor/index.tsx#L41

For others landing on this issue, feel free to add more context or simply leave a thumbs up 👍 on this comment if you find this feature useful.
